src/pages/app.jsx

```diff
@@ -62,123 +78,38 @@ const App = React.createClass({
     SessionStore.setRoutes(routes);
   },
 
-  renderContent() {
-    const styles = {
-      display: 'flex',
-      flexDirection: 'column',
-      flex: 1
-    };
-
-    return (
-      <Breakpoint
-        minWidth={768}
-        widthMethod="componentWidth"
-        style={styles}
-      >
-        {this.props.children}
-        <SnackbarNotification />
-      </Breakpoint>
-    );
-  },
+  renderHelmet() {
```

Comment:
do we need a separate method to render a single component?
(comment isn't necessary i removed this dependency in #205